src/objects/ProductionDbConfig.js

```diff
@@ -43,15 +41,19 @@ class ProductionDbConfig {
       console.error(error);
     });
   }
-  
+
   async for_(database) {
     let config = this.configs[database.toLowerCase()];
     if (!config) {
       this.addConfigs(await this._fetchDatasources());
     }
-    
+
     return this.configs[database.toLowerCase()];
   }
+
+  toString () {
+    return this.dbConfig.toString();
```

Comment:
this doesn't seem like the right change. 
Both `StagingDbConfig` and `ProductionDbConfig` classes' `for_` methods return a `dbConfig` object which then needs to be outputted to `string`. This method will always return the `string` for this instance's `dbConfig` object, and not the one returned by the `for_` method. That's why the `toString` method needs to be on the `DbConfig` class. 

Maybe you can rename your variable when consuming all of this so it doesn't read so redundant.